app/api/events/$id.mjs

```diff
@@ -1,9 +1,38 @@
-import { events } from '../events.mjs'
+import events from '../../data/events.json' assert { type: 'json' }
+import sponsors from '../../data/sponsors.json' assert { type: 'json' }
+import talks from '../../data/talks.json' assert { type: 'json' }
+import speakers from '../../data/speakers.json' assert { type: 'json' }
+
+function inflateTalk(talk_id) {
+  let talk = talks.find(t => t.id === talk_id)
+  if (talk && talk.speaker_id) {
+    talk.speaker = speakers.find(s => s.id === talk.speaker_id)
+  }
+  return talk
+}
+
+function inflateSponsors(sponsor_id) {
+  return sponsors.find(s => s.id === sponsor_id)
+}
+
+function inflateEvent(event) {
+  if (event.talks) {
+    event.talks = event.talks.map(inflateTalk)
+    event.sponsors = event.sponsors.map(inflateSponsors)
+  }
+  return event
+}
```

Comment:
Instead of copying these functions from api/index.mjs, let's make them helpers in their respective files. i.e. let's put `inflateTalk` as an exported helper function in api/talks.mjs that api/index.mjs and api/events/$id.mjs can both import and use.

Same with the other helpers for api/events.mjs and api/sponsors.mjs

Reply:
✅